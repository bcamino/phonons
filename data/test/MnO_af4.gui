   3   1   1
  0.000000000000E+00 -0.313880000000E+01  0.000000000000E+00
 -0.443890000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00 -0.627750000000E+01
    8
  0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00  0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00  0.100000000000E+01
  0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
 -0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00 -0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00  0.100000000000E+01
 -0.221945000000E+01 -0.156940000000E+01  0.000000000000E+00
 -0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00  0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00 -0.100000000000E+01
  0.000000000000E+00 -0.156940000000E+01  0.000000000000E+00
  0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00 -0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00 -0.100000000000E+01
 -0.221945000000E+01  0.000000000000E+00  0.000000000000E+00
 -0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00 -0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00 -0.100000000000E+01
  0.000000000000E+00  0.000000000000E+00  0.000000000000E+00
  0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00  0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00 -0.100000000000E+01
 -0.221945000000E+01 -0.156940000000E+01  0.000000000000E+00
  0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00 -0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00  0.100000000000E+01
  0.000000000000E+00 -0.156940000000E+01  0.000000000000E+00
 -0.100000000000E+01  0.000000000000E+00  0.000000000000E+00
  0.000000000000E+00  0.100000000000E+01  0.000000000000E+00
  0.000000000000E+00  0.000000000000E+00  0.100000000000E+01
 -0.221945000000E+01  0.000000000000E+00  0.000000000000E+00
    8
   56     -3.329175000000     -2.354100000000     -2.354062500000
   56     -1.109725000000     -0.784700000000     -3.923437500000
   20     -3.329175000000     -2.354100000000     -5.492812500000
   20     -1.109725000000     -0.784700000000     -0.784687500000
    8     -3.329175000000     -0.784700000000     -3.923437500000
    8     -1.109725000000     -2.354100000000     -2.354062500000
    8     -3.329175000000     -0.784700000000     -0.784687500000
    8     -1.109725000000     -2.354100000000     -5.492812500000
   59    8
