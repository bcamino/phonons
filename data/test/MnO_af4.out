date Thu 12 Jan 2023 14:57:14 GMT
hostname MacBook-Pro-8.local
system Darwin MacBook-Pro-8.local 21.6.0 Darwin Kernel Version 21.6.0: Mon Aug 22 20:20:05 PDT 2022; root:xnu-8020.140.49~2/RELEASE_ARM64_T8101 arm64
user brunocamino
output data in /Users/brunocamino/Desktop/UCL/crystal/phonons/data/test/MnO_af4.out
crystal executable in /Users/brunocamino/crystal
input data in /Users/brunocamino/Desktop/UCL/crystal/phonons/data/test/MnO_af4.d12
temporary directory /Users/brunocamino/crystal/tmp
external geometry input from /Users/brunocamino/Desktop/UCL/crystal/phonons/data/test/MnO_af4.gui
external geometry input saved in /Users/brunocamino/Desktop/UCL/crystal/phonons/data/test/MnO_af4_inp.gui
test
EXTERNAL
EXTPRT
TESTGEOM
END

         
 *******************************************************************************
 *                                                                             *
 *                               CRYSTAL17                                     *
 *                      public : 1.0.1 - Feb 1st, 2017                         *
 *                       HTTP://WWW.CRYSTAL.UNITO.IT                           *
 *                                                                             *
 *                              MAIN AUTHORS                                   *
 *                                                                             *
 *  R. DOVESI(1,11), V.R. SAUNDERS(2), C. ROETTI(1,11), R. ORLANDO(1,11),      *
 *  C.M. ZICOVICH-WILSON(1,3), F. PASCALE(4), B. CIVALLERI(1,11),              *
 *  K. DOLL(5), N.M. HARRISON(2,6), I.J. BUSH(7), Ph. D'ARCO(8),               *
 *  M. LLUNELL(9), M. CAUSA'(10), Y. NOEL(8), L. MASCHIO(1,11), A. ERBA(1,11)  *
 *  M. RE'RAT(14), S. CASASSA(1,11)                                            *
 *                                                                             *
 *        CONTRIBUTIONS TO THE CODE HAVE ALSO BEEN GIVEN BY                    *
 *                                                                             *
 *  M. FERRABONE(1,11), M. DE LA PIERRE(1,11), M. FERRERO(1,11),               *
 *  J. BAIMA(1,11), E. ALBANESE(1,11), M.F. PEINTINGER(12), R. BAST(13),       *
 *  B. KIRTMAN(15), V. LACIVITA(1,11), R. DEMICHELIS(1,16)                     *
 *  G. BRANDEBURG(17), S. SALUSTRO(1,11), G. SANSONE(1,11)                     *
 *  M.T. RUGGIERO(18)                                                          *
 *                                                                             *
 * (1) THEORETICAL CHEMISTRY GROUP - UNIVERSITA' DI TORINO - TORINO (ITALY)    *

 * (2) COMPUTATIONAL SCIENCE & ENGINEERING DEPARTMENT - STFC DARESBURY (UK)    *
 * (3) UNIVERSIDAD AUTONOMA DEL ESTADO DE MORELOS - CUERNAVACA (MEXICO)        *
 * (4) UNIVERSITE' DE LORRAINE - NANCY (FRANCE)                                *
 * (5) UNIVERSITAET STUTTGART- STUTTGART (GERMANY)                             *
 * (6) IMPERIAL COLLEGE - LONDON (UK)                                          *
 * (7) OXFORD E-RESEARCH CENTRE - OXFORD (UK)                                  *
 * (8) UPMC - SORBONNE UNIVERSITES - PARIS (FRANCE)                            *
 * (9) UNIVERSIDAD DE BARCELONA - BARCELONA (SPAIN)                            *
 *(10) UNIVERSITA' DI NAPOLI "FEDERICO II" - NAPOLI (ITALY)                    *
 *(11) NIS - NANOSTRUCTURED INTERFACES AND SURFACES - TORINO (ITALY)           *
 *(12) MPI FUER CHEMISCHE ENERGIEKONVERSION - MUELHEIM AN DER RUHR (GERMANY)   *
 *(13) UNIVERSITY OF TROMSO - TROMSO (NORWAY)                                  *
 *(14) UNIVERSITE' DE PAU ET DES PAYS DE L'ADOUR - PAU (FRANCE)                *
 *(15) UNIVERSITY OF CALIFORNIA SANTA BARBARA - SANTA BARBARA (USA)            *
 *(16) CURTIN UNIVERSITY - PERTH (AUSTRALIA)                                   *
 *(17) UNIVERSITY COLLEGE LONDON - LONDON (UK)                                 *
 *(18) CAMBRIDGE UNIVERSITY - CAMBRIDGE (UK)                                   *
 *                                                                             *
 *******************************************************************************
 EEEEEEEEEE STARTING  DATE 12 01 2023 TIME 14:57:14.1
 test                                                                            

 GEOMETRY INPUT FROM EXTERNAL FILE (FORTRAN UNIT 34)
 3D - CRYSTAL 

 *******************************************************************************
 LATTICE PARAMETERS  (ANGSTROMS AND DEGREES) - PRIMITIVE CELL
       A          B          C        ALPHA      BETA     GAMMA        VOLUME
    3.13880    4.43890    6.27750    90.00000  90.00000  90.00000     87.463273

 NUMBER OF IRREDUCIBLE ATOMS IN THE CELL:    4

 CARTESIAN COORDINATES (ANGSTROM) - IRREDUCIBLE ATOMS AND SYMMETRY RELATED

 AT.IRR.  AT  AT.N.          X                   Y                   Z
     1     1    56   -3.329175000000E+00 -2.354100000000E+00 -2.354062500000E+00
     1     2    56   -1.109725000000E+00 -7.847000000000E-01 -3.923437500000E+00
     3     3    20   -3.329175000000E+00 -2.354100000000E+00 -5.492812500000E+00
     3     4    20   -1.109725000000E+00 -7.847000000000E-01 -7.846875000000E-01
     5     5     8   -3.329175000000E+00 -7.847000000000E-01 -3.923437500000E+00
     5     6     8   -1.109725000000E+00 -2.354100000000E+00 -2.354062500000E+00
     7     7     8   -3.329175000000E+00 -7.847000000000E-01 -7.846875000000E-01
     7     8     8   -1.109725000000E+00 -2.354100000000E+00 -5.492812500000E+00

 SPACE  GROUP  N.                     :   59
 NUMBER OF SYMMETRY OPERATORS         :    8
 *******************************************************************************
 LATTICE PARAMETERS (ANGSTROMS AND DEGREES) - BOHR = 0.5291772083 ANGSTROM
 PRIMITIVE CELL - CENTRING CODE 1/0 VOLUME=    87.463273 - DENSITY  7.969 g/cm^3
         A              B              C           ALPHA      BETA       GAMMA 
     3.13880000     4.43890000     6.27750000    90.000000  90.000000  90.000000
 *******************************************************************************
 ATOMS IN THE ASYMMETRIC UNIT    4 - ATOMS IN THE UNIT CELL:    8
     ATOM                 X/A                 Y/B                 Z/C    
 *******************************************************************************
      1 T  56 BA   -2.500000000000E-01 -2.500000000000E-01  3.750000000000E-01
      2 F  56 BA    2.500000000000E-01  2.500000000000E-01 -3.750000000000E-01
      3 T  20 CA   -2.500000000000E-01 -2.500000000000E-01 -1.250000000000E-01
      4 F  20 CA    2.500000000000E-01  2.500000000000E-01  1.250000000000E-01
      5 T   8 O     2.500000000000E-01 -2.500000000000E-01 -3.750000000000E-01
      6 F   8 O    -2.500000000000E-01  2.500000000000E-01  3.750000000000E-01
      7 T   8 O     2.500000000000E-01 -2.500000000000E-01  1.250000000000E-01
      8 F   8 O    -2.500000000000E-01  2.500000000000E-01 -1.250000000000E-01

 T = ATOM BELONGING TO THE ASYMMETRIC UNIT
 *******************************************************************************
 * GEOMETRY EDITING - INPUT COORDINATES ARE GIVEN IN ANGSTROM         
 *******************************************************************************

 GEOMETRY NOW FULLY CONSISTENT WITH THE GROUP

 GCALCO - MAX INDICES DIRECT LATTICE VECTOR    21    15    10
 NO.OF VECTORS CREATED 6999 STARS 1056 RMAX    99.25189 BOHR

 GEOMETRY FOR WAVE FUNCTION - DIMENSIONALITY OF THE SYSTEM    3
 (NON PERIODIC DIRECTION: LATTICE PARAMETER FORMALLY SET TO 500)
 *******************************************************************************
 LATTICE PARAMETERS (ANGSTROMS AND DEGREES) - BOHR = 0.5291772083 ANGSTROM
 PRIMITIVE CELL - CENTRING CODE 1/0 VOLUME=    87.463273 - DENSITY  7.969 g/cm^3
         A              B              C           ALPHA      BETA       GAMMA 
     3.13880000     4.43890000     6.27750000    90.000000  90.000000  90.000000
 *******************************************************************************
 ATOMS IN THE ASYMMETRIC UNIT    4 - ATOMS IN THE UNIT CELL:    8
     ATOM                 X/A                 Y/B                 Z/C    
 *******************************************************************************
      1 T  56 BA   -2.500000000000E-01 -2.500000000000E-01  3.750000000000E-01
      2 F  56 BA    2.500000000000E-01  2.500000000000E-01 -3.750000000000E-01
      3 T  20 CA   -2.500000000000E-01 -2.500000000000E-01 -1.250000000000E-01
      4 F  20 CA    2.500000000000E-01  2.500000000000E-01  1.250000000000E-01
      5 T   8 O     2.500000000000E-01 -2.500000000000E-01 -3.750000000000E-01
      6 F   8 O    -2.500000000000E-01  2.500000000000E-01  3.750000000000E-01
      7 T   8 O     2.500000000000E-01 -2.500000000000E-01  1.250000000000E-01
      8 F   8 O    -2.500000000000E-01  2.500000000000E-01 -1.250000000000E-01

 T = ATOM BELONGING TO THE ASYMMETRIC UNIT

 ****    8 SYMMOPS - TRANSLATORS IN FRACTIONAL UNITS
 **** MATRICES AND TRANSLATORS IN THE CRYSTALLOGRAPHIC REFERENCE FRAME
   V INV                    ROTATION MATRICES                   TRANSLATORS
   1   1  1.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00
   2   2 -1.00  0.00  0.00  0.00 -1.00  0.00  0.00  0.00  1.00  0.50  0.50  0.00
   3   3  1.00  0.00  0.00  0.00 -1.00  0.00  0.00  0.00 -1.00  0.50  0.00  0.00
   4   4 -1.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00 -1.00  0.00  0.50  0.00
   5   5 -1.00  0.00  0.00  0.00 -1.00  0.00  0.00  0.00 -1.00  0.00  0.00  0.00
   6   6  1.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00 -1.00  0.50  0.50  0.00
   7   7 -1.00  0.00  0.00  0.00  1.00  0.00  0.00  0.00  1.00  0.50  0.00  0.00
   8   8  1.00  0.00  0.00  0.00 -1.00  0.00  0.00  0.00  1.00  0.00  0.50  0.00

 DIRECT LATTICE VECTORS CARTESIAN COMPONENTS (ANGSTROM)
          X                    Y                    Z
   0.000000000000E+00  -0.313880000000E+01   0.000000000000E+00
  -0.443890000000E+01   0.000000000000E+00   0.000000000000E+00
   0.000000000000E+00   0.000000000000E+00  -0.627750000000E+01


 CARTESIAN COORDINATES - PRIMITIVE CELL
 *******************************************************************************
 *      ATOM          X(ANGSTROM)         Y(ANGSTROM)         Z(ANGSTROM)
 *******************************************************************************
    1    56 BA   -3.329175000000E+00 -2.354100000000E+00 -2.354062500000E+00
    2    56 BA   -1.109725000000E+00 -7.847000000000E-01 -3.923437500000E+00
    3    20 CA   -3.329175000000E+00 -2.354100000000E+00 -5.492812500000E+00
    4    20 CA   -1.109725000000E+00 -7.847000000000E-01 -7.846875000000E-01
    5     8 O    -3.329175000000E+00 -7.847000000000E-01 -3.923437500000E+00
    6     8 O    -1.109725000000E+00 -2.354100000000E+00 -2.354062500000E+00
    7     8 O    -3.329175000000E+00 -7.847000000000E-01 -7.846875000000E-01
    8     8 O    -1.109725000000E+00 -2.354100000000E+00 -5.492812500000E+00
 WARNING **** CRYSTA **** TESTGEOM  DIRECTIVE - GEOMETRY EDITING COMPLETE
 TTTTTTTTTTTTTTTTTTTTTTTTTTTTTT END         TELAPSE        0.02 TCPU        0.01
 EEEEEEEEEE TERMINATION  DATE 12 01 2023 TIME 14:57:14.1
Thu 12 Jan 2023 14:57:14 GMT
Contents of temporary directory:
total 8
-rw-r--r--  1 brunocamino  admin  2699 12 Jan 14:57 fort.34
SCF abnormal end - no wf written in fort.9
geometry input file fort.34 saved as /Users/brunocamino/Desktop/UCL/crystal/phonons/data/test/MnO_af4.gui
temporary directory /Users/brunocamino/crystal/tmp removed
